User Interface/Tooltips/.gitignore

```diff
@@ -0,0 +1,5 @@
+/flixel/
```

Comment:
Do you need this .gitignore for anything? export folders are already ignored in the gitignore in the root directory. Same goes for RPG Interface I guess, since it seems to have been copied from there.